pages/enterprise/cloud_forwarder/cloud_forwarder.rst

```diff
@@ -0,0 +1,415 @@
+        }
+
+Input Endpoint
+--------------
+
+To obtain a list of Inputs running on the Forwarder, query the endpoint ``GET /api/inputs``.
+
+    .. code-block:: json
+
+        {
+            "inputs": [
+                {
+                    "id": "5fc91564d44bfd2000249e8c",
+                    "title": "Random"
+                },
+                {
+                    "id": "5fc91550d44bfd2000249e74",
+                    "title": "Beats"
+                }
+            ]
+        }   
+
+Drill down to the input profile and view the Forwarder sub-menu to ensure it receives messages. If data still doesn’t 
+come through, create a new input. Click the name of your input which takes you to its main profile with the details 
+you added at initial configuration.
+        
+        
+If an input is in a failed state, the input endpoint returns no information on ``id`` nor ``title``. 
+        
+Additionally, you can get deeper insights into Forwarder messaging and node health. Click the Details button on your 
+node, to get information about your message cache (buffers). One metric to pay attention to is the number of messages 
+in the journal. The journal is the on-disk persistent storage location that stores Forwarder messages. So, if the rest 
+of Graylog malfunctions, we can still keep all the messages we still have a place to put them.
+        
+Prometheus Metrics Exports
+--------------------------
+
+The Forwarder, alone, has no interface for insight into the internal operations. To that end, you must configure a 
+local Prometheus container; this becomes the interface for Forwarder metrics. These are similar to the traditional 
+Graylog Server metrics but instead are exported to Prometheus. The response format is the standard Prometheus HTTP 
+export format.
+        
+To start this process:
+
+#. Download and start `Prometheus <https://prometheus.io/docs/prometheus/latest/getting_started/#downloading-and-running-prometheus>`__.
+#. Install `Docker <https://docs.docker.com/get-docker/>`__ on your machine.
+#. Create a Prometheus Dockerfile, e.g. ``touch /tmp/prometheus.yml``:
+    
+    .. code-block:: yaml
+        
+        $ global:
+          scrape_interval: 15s
+          scrape_timeout: 10s
+          evaluation_interval: 15s
+        alerting:
+          alertmanagers:
+          - static_configs:
+            - targets: []
+            scheme: http
+            timeout: 10s
+            api_version: v1
+        scrape_configs:
+        - job_name: prometheus
+          honor_timestamps: true
+          scrape_interval: 15s
+          scrape_timeout: 10s
+          metrics_path: /api/metrics/prometheus
+          scheme: http
+          static_configs:
+          - targets:
+            - host.docker.internal:9001
+        Run this Docker command to start the container:
+            docker run \
+              -p 9090:9090 \
+              -v /tmp/prometheus.yml:/etc/prometheus/prometheus.yml \
+              prom/prometheus
```

Comment:
We also need this to avoid embedding of the "Run this Docker ..." into the code block:

```suggestion

Run this Docker command to start the container:

    .. code-block:: bash

            docker run \
              -p 9090:9090 \
              -v /tmp/prometheus.yml:/etc/prometheus/prometheus.yml \
              prom/prometheus
```